evm.md

```diff
@@ -1292,11 +1292,23 @@ For now, I assume that they instantiate an empty account and use the empty data.
          <account>
            <acctID> ACCT </acctID>
            <code> CODE </code>
+           <nonce> NONCE </nonce>
+           <balance> BAL </balance>
            ...
          </account>
+      requires notBool #accountEmpty(CODE, NONCE, BAL)
 ```
 
 ```k
+     rule <k> EXTCODEHASH ACCT => 0 ~> #push ... </k>
+         <account>
+           <acctID> ACCT </acctID>
+           <code> .WordStack </code>
+           <nonce> 0 </nonce>
+           <balance> 0 </balance>
+           ...
+         </account>
```

Comment:
Should we be using `#accountNonexistent` instead? The question has two components:

-   Should we take into account the schedule constant `Gemptyisnonexistent` for this check?
-   Do we want to fold this rule and the following rule into one rule?